File: frontend/app/src/main/java/com/example/fastfashion/model/FashionItemCreate.kt
package com.example.fastfashion.model

import com.google.gson.annotations.SerializedName

data class FashionItemCreate (

    @SerializedName("type")
    val type: String,
    @SerializedName("detail")
    val detail: String,
    @SerializedName("style")
    val style: String,
    @SerializedName("date")
    val date: String,
    @SerializedName("pictureUri")
    val uri: String,
    @SerializedName("userId")
    val userId:Int
)

File: backend/FastFashion/Models/FashionItemCreate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FastFashion.Models
{
    public class FashionItemCreate
    {

       
        public string Type { get; set; }

        public string Detail { get; set; }

        public string Style { get; set; }

        public string Date { get; set; }

        public string pictureUri { get; set; }

        public int UserId { get; set; }

    }
}
